src/components/Loader.tsx

```diff
@@ -1,8 +1,9 @@
 import Spinner from "react-native-spinkit";
 import React from "react";
+import {colors} from "../../assets/colors/colors";
 
 export const Loader = () => {
     return (
-        <Spinner type={"WanderingCubes"} style={{alignSelf: "center", justifyContent: "center", alignItems: "center", flex: 1}} size={150} color={"#4b37bc"}/>
+        <Spinner type={"WanderingCubes"} style={{alignSelf: "center", justifyContent: "center", alignItems: "center", flex: 1}} size={150} color={colors.BLUE}/>
```

Comment:
We should never pass style props as an object like this. Use https://archive.reactnative.dev/docs/0.57/stylesheet